src/script/page/message-list/MentionSuggestions/MentionSuggestionsItem.tsx

```diff
@@ -43,7 +43,7 @@ const MentionSuggestionsItem: React.ForwardRefRenderFunction<HTMLDivElement, Men
     ['name', 'expirationRemainingText', 'isTemporaryGuest', 'isExternal', 'isDirectGuest'],
   );
 
-  const onClick = (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+  const onClick = (event: React.MouseEvent<HTMLDivElement, MouseEvent> | React.KeyboardEvent<HTMLDivElement>) => {
```

Comment:
```suggestion
  const onClick = (event: React.UIEvent) => {
```
As we never use any specificity of `Keyboard` or `Mouse` event, we can go for a more generic type

Reply:
Sure, didn't know that such a type exists, I was always explicitly typing it. Makes a lot of sense.

Reply:
I've checked it and we're always passing an array to props (sometimes empty), so definitely no reason to make it optional here ;)